docs/articles/documentation/test-api/authentication/user-roles.md

```diff
@@ -131,4 +131,28 @@ TestCafe will navigate to the saved URL each time after you switch to this role.
 
 This option is useful if you store session-related data (like session ID) in the URL.
 
+```js
+import { Role } from 'testcafe';
+
+const role = Role('http://example.com/login', async t => {
+    await t
+        .typeText('#login', 'username')
+        .typeText('#password', 'password')
+        .click('#sign-in'); // Redirects to http://example.com?sessionId=abcdef
+}, {
```

Comment:
IMHO we can just use 
```
}, { preserveUrl: true });
``` 
here

Reply:
I feel like the way it is, this option is emphasized a bit (at least). The way you suggest - it could be lost behind a larger code block.